src/components/Signup/Signup.test.js

```diff
@@ -5,21 +5,33 @@ import DopplerIntlProvider from '../../i18n/DopplerIntlProvider';
 import { MemoryRouter as Router } from 'react-router-dom';
 import { AppServicesProvider } from '../../services/pure-di';
 import '@testing-library/jest-dom/extend-expect';
+import { HardcodedDopplerSitesClient } from '../../services/doppler-sites-client.doubles';
+import { timeout } from '../../utils';
 
 class MenubuttonFake {
   init() {}
 }
 
+const emptyResponse = { success: false, error: new Error('Dummy error') };
+
+const defaultDependencies = {
```

Comment:
Perhaps this default dependencies could be in a separate file to be reused by multiple tests. What do you think?

Reply:
Yes is the same comment to said @andresmoschini we can add in another pr to reuse in more test and now in this pr only fix the errors